AI/finder/ticker_selector.py

```
@@ -0,0 +1,185 @@
+
+
+# --- private 함수 ---
+
+def _parse_summary(x):
+    '''
+    JSON 문자열을 dict로 변환해주는 함수
+    '''
+    if isinstance(x, str):
+        try:
+            return json.loads(x)
+        except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)